outport/host/errors.go

```diff
@@ -7,3 +7,6 @@ var ErrHostIsClosed = errors.New("server is closed")
 
 // ErrNilHost signals that a nil host has been provided
 var ErrNilHost = errors.New("nil host provided")
+
+// ErrNilStateChangesCollector signals that a nil state change collector has been provided.
+var ErrNilStateChangesCollector = errors.New("nil state changes collector provided")
```

Comment:
This is not used anywhere.

Reply:
it is used here: state/accountsDB.go:163